sktm/patchwork.py

```diff
@@ -483,7 +483,7 @@ def get_patch_by_id(self, pid):
 
         return r.json()
 
-    def get_patchsets_by_patch(self, url, db=None, seen=set()):
+    def get_patchsets_by_patch(self, url, seen=set()):
```

Comment:
Do you want to have this in this pull or should I put it into my patchwork pull?

Reply:
This was just a side-effect of me considering if we should access database in Patchwork, and seeing some stuff there I thought we'd better distance ourselves from that idea. It's not at all needed for adding filtering, so if it's better for you, take it over, and I'll remove it from here :)

Reply:
That's what I thought :) I'll add it to the pull.